src/Service/ServiceManagerConfig.php

```diff
@@ -64,37 +64,29 @@ class ServiceManagerConfig extends Config
      */
     public function __construct(array $config = [])
     {
-        $this->config['factories']['ServiceManager'] = function ($container) {
-            return $container;
-        };
+        $this->config['factories']['ServiceManager'] = static fn($container) => $container;
 
-        $this->config['factories']['SharedEventManager'] = function () {
-            return new SharedEventManager();
-        };
+        $this->config['factories']['SharedEventManager'] = static fn(): SharedEventManager => new SharedEventManager();
 
         $this->config['initializers'] = ArrayUtils::merge($this->config['initializers'], [
-            'EventManagerAwareInitializer' => function ($first, $second) {
+            'EventManagerAwareInitializer' => static function ($first, $second) : void {
                 if ($first instanceof ContainerInterface) {
                     $container = $first;
                     $instance = $second;
                 } else {
                     $container = $second;
                     $instance = $first;
                 }
-
```

Comment:
I really don't like that rector gets rid of random whitespace, but it's not critical :)

Reply:
that's on PHP-Parser latest dev-master for keep the space, but probably only will land on PHP-Parser 5.0 https://github.com/nikic/PHP-Parser/pull/891

Reply:
Cool!